src/components/ui/FiltersPart.tsx

```
@@ -0,0 +1,194 @@
+
+          <div className="space-y-2">
+            <h3 className="text-sm font-semibold text-gray-900">Condition</h3>
+            {conditions.map(c => <Checkbox key={c} label={c} checked={filters.condition.includes(c)} onChange={() => toggle("condition", c)} />)}
+          </div>
+
+          <div className="space-y-2">
+            <h3 className="text-sm font-semibold text-gray-900">Price Range</h3>
+            <div className="flex gap-2">
+              <div className="flex-1">
+                <label className="block text-xs text-gray-600 mb-1">From</label>
+                <input type="number" placeholder="0 ILS" value={filters.priceRange.min} onChange={e => update("priceRange", { ...filters.priceRange, min: e.target.value })} className="w-full px-3 py-2 border border-gray-300 rounded-lg text-sm" />
```

Biome (2.3.13)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)